docs/source/install.rst

```diff
@@ -25,8 +23,8 @@ the IPython kernel.
     document.
 
 
-How to Install Jupyter Notebook
--------------------------------
+Quickstart
```

Comment:
In my mind "quickstart" or "getting started" includes both installation and running the notebook. I thought about putting that as a heading above those two pages, but I think that is too much hierarchy at this level. Based on this I would just leave it as it was. Also, our previous Google Analytics showed that "install" was the main phrase people were looking for.